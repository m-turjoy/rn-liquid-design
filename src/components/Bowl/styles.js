import { StyleSheet } from 'react-native'

const styles = StyleSheet.create({
  waveBall: {
    width: 100,
    aspectRatio: 1,
    overflow: 'hidden'
  },
  icon: {
    width: 24,
    height: 24,
    position: 'absolute',
    left: 38,
    bottom: 36,
    justifyContent: 'center',
    alignItems: 'center'
  },
  iconWrapper: {
    width: 140,
    height: 140,
    position: 'absolute',
    left: -20,
    bottom: -20
  }
})

export default styles
