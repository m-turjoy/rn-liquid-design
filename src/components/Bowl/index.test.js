import React from 'react'
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Bowl from './index'

configure({ adapter: new Adapter() })
describe('Bowl component', () => {
  describe('Renders correctly', () => {
    test('it renders default Bowl', () => {
      const tree = shallow(
        <Bowl
          H={70}
        />
      )
      expect(tree).toMatchSnapshot()
    })
  })
})
