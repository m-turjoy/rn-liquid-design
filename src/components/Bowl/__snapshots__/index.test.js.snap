// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bowl component Renders correctly it renders default Bowl 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Bowl
    H={70}
    looped={false}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <View
          style={
            Object {
              "aspectRatio": 1,
              "overflow": "hidden",
              "width": 100,
            }
          }
        >
          <AnimatedComponent
            preserveAspectRatio="xMinYMin meet"
            style={
              Object {
                "bottom": -23,
                "height": 30,
                "left": 0,
                "position": "absolute",
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                  Object {
                    "translateY": 0,
                  },
                ],
                "width": 420,
              }
            }
            viewBox="0 0 280 20"
          >
            <Path
              d="M 0 0
                   Q 35 -10 70 0
                   T 140 0
                   T 210 0
                   T 280 0
                   T 350 0
                   T 420 0
                   V 100
                   H 0
                   Z"
              transform="translate(0, 10)"
            />
          </AnimatedComponent>
          <AnimatedComponent
            preserveAspectRatio="xMinYMin meet"
            style={
              Object {
                "bottom": -23,
                "height": 30,
                "left": 0,
                "position": "absolute",
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                  Object {
                    "translateY": 0,
                  },
                ],
                "width": 540,
              }
            }
            viewBox="0 0 360 25"
          >
            <Path
              d="M 0 0
                   Q 45 -15 90 0
                   T 180 0
                   T 270 0
                   T 360 0
                   T 450 0
                   T 540 0
                   V 100
                   H 0
                   Z"
              transform="translate(0, 15)"
            />
          </AnimatedComponent>
          <Svg
            preserveAspectRatio="xMidYMid meet"
            style={
              Object {
                "bottom": -20,
                "height": 140,
                "left": -20,
                "position": "absolute",
                "width": 140,
              }
            }
          >
            <Circle
              cx="70"
              cy="70"
              fill="transparent"
              r="60"
              stroke="#ffffff"
              strokeWidth={24}
            />
          </Svg>
          <AnimatedComponent
            style={
              Object {
                "bottom": 0,
                "height": 100,
                "left": 0,
                "position": "absolute",
                "transform": Array [
                  Object {
                    "translateY": 0,
                  },
                ],
                "width": 100,
              }
            }
          >
            <AnimatedComponent
              cx={80}
              fill="rgba(255,255,255, 0.3)"
              r="5"
            />
          </AnimatedComponent>
          <AnimatedComponent
            style={
              Object {
                "bottom": 0,
                "height": 100,
                "left": 0,
                "position": "absolute",
                "transform": Array [
                  Object {
                    "translateY": 0,
                  },
                ],
                "width": 100,
              }
            }
          >
            <AnimatedComponent
              cx={70}
              fill="rgba(255,255,255, 0.3)"
              r="3"
            />
          </AnimatedComponent>
        </View>,
        <Svg
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "bottom": -20,
              "height": 140,
              "left": -20,
              "position": "absolute",
              "width": 140,
            }
          }
        >
          <Circle
            cx="70"
            cy="70"
            fill="transparent"
            r="55"
            stroke="#efeff1"
            strokeWidth={4}
          />
        </Svg>,
        <AnimatedComponent
          style={
            Object {
              "alignItems": "center",
              "bottom": 36,
              "height": 24,
              "justifyContent": "center",
              "left": 38,
              "position": "absolute",
              "width": 24,
            }
          }
        >
          <Icon
            color="red"
            name="attention1"
            size={24}
            themeName="vibrantCyan"
          />
        </AnimatedComponent>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            Array [
              <AnimatedComponent
                preserveAspectRatio="xMinYMin meet"
                style={
                  Object {
                    "bottom": -23,
                    "height": 30,
                    "left": 0,
                    "position": "absolute",
                    "transform": Array [
                      Object {
                        "translateX": 0,
                      },
                      Object {
                        "translateY": 0,
                      },
                    ],
                    "width": 420,
                  }
                }
                viewBox="0 0 280 20"
              >
                <Path
                  d="M 0 0
                   Q 35 -10 70 0
                   T 140 0
                   T 210 0
                   T 280 0
                   T 350 0
                   T 420 0
                   V 100
                   H 0
                   Z"
                  transform="translate(0, 10)"
                />
              </AnimatedComponent>,
              <AnimatedComponent
                preserveAspectRatio="xMinYMin meet"
                style={
                  Object {
                    "bottom": -23,
                    "height": 30,
                    "left": 0,
                    "position": "absolute",
                    "transform": Array [
                      Object {
                        "translateX": 0,
                      },
                      Object {
                        "translateY": 0,
                      },
                    ],
                    "width": 540,
                  }
                }
                viewBox="0 0 360 25"
              >
                <Path
                  d="M 0 0
                   Q 45 -15 90 0
                   T 180 0
                   T 270 0
                   T 360 0
                   T 450 0
                   T 540 0
                   V 100
                   H 0
                   Z"
                  transform="translate(0, 15)"
                />
              </AnimatedComponent>,
            ],
            <Svg
              preserveAspectRatio="xMidYMid meet"
              style={
                Object {
                  "bottom": -20,
                  "height": 140,
                  "left": -20,
                  "position": "absolute",
                  "width": 140,
                }
              }
            >
              <Circle
                cx="70"
                cy="70"
                fill="transparent"
                r="60"
                stroke="#ffffff"
                strokeWidth={24}
              />
            </Svg>,
            <AnimatedComponent
              style={
                Object {
                  "bottom": 0,
                  "height": 100,
                  "left": 0,
                  "position": "absolute",
                  "transform": Array [
                    Object {
                      "translateY": 0,
                    },
                  ],
                  "width": 100,
                }
              }
            >
              <AnimatedComponent
                cx={80}
                fill="rgba(255,255,255, 0.3)"
                r="5"
              />
            </AnimatedComponent>,
            <AnimatedComponent
              style={
                Object {
                  "bottom": 0,
                  "height": 100,
                  "left": 0,
                  "position": "absolute",
                  "transform": Array [
                    Object {
                      "translateY": 0,
                    },
                  ],
                  "width": 100,
                }
              }
            >
              <AnimatedComponent
                cx={70}
                fill="rgba(255,255,255, 0.3)"
                r="3"
              />
            </AnimatedComponent>,
          ],
          "style": Object {
            "aspectRatio": 1,
            "overflow": "hidden",
            "width": 100,
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "0",
            "nodeType": "class",
            "props": Object {
              "children": <Path
                d="M 0 0
                   Q 35 -10 70 0
                   T 140 0
                   T 210 0
                   T 280 0
                   T 350 0
                   T 420 0
                   V 100
                   H 0
                   Z"
                transform="translate(0, 10)"
              />,
              "preserveAspectRatio": "xMinYMin meet",
              "style": Object {
                "bottom": -23,
                "height": 30,
                "left": 0,
                "position": "absolute",
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                  Object {
                    "translateY": 0,
                  },
                ],
                "width": 420,
              },
              "viewBox": "0 0 280 20",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "d": "M 0 0
                   Q 35 -10 70 0
                   T 140 0
                   T 210 0
                   T 280 0
                   T 350 0
                   T 420 0
                   V 100
                   H 0
                   Z",
                "transform": "translate(0, 10)",
              },
              "ref": [Function],
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "1",
            "nodeType": "class",
            "props": Object {
              "children": <Path
                d="M 0 0
                   Q 45 -15 90 0
                   T 180 0
                   T 270 0
                   T 360 0
                   T 450 0
                   T 540 0
                   V 100
                   H 0
                   Z"
                transform="translate(0, 15)"
              />,
              "preserveAspectRatio": "xMinYMin meet",
              "style": Object {
                "bottom": -23,
                "height": 30,
                "left": 0,
                "position": "absolute",
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                  Object {
                    "translateY": 0,
                  },
                ],
                "width": 540,
              },
              "viewBox": "0 0 360 25",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "d": "M 0 0
                   Q 45 -15 90 0
                   T 180 0
                   T 270 0
                   T 360 0
                   T 450 0
                   T 540 0
                   V 100
                   H 0
                   Z",
                "transform": "translate(0, 15)",
              },
              "ref": [Function],
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <Circle
                cx="70"
                cy="70"
                fill="transparent"
                r="60"
                stroke="#ffffff"
                strokeWidth={24}
              />,
              "preserveAspectRatio": "xMidYMid meet",
              "style": Object {
                "bottom": -20,
                "height": 140,
                "left": -20,
                "position": "absolute",
                "width": 140,
              },
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "cx": "70",
                "cy": "70",
                "fill": "transparent",
                "r": "60",
                "stroke": "#ffffff",
                "strokeWidth": 24,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <AnimatedComponent
                cx={80}
                fill="rgba(255,255,255, 0.3)"
                r="5"
              />,
              "style": Object {
                "bottom": 0,
                "height": 100,
                "left": 0,
                "position": "absolute",
                "transform": Array [
                  Object {
                    "translateY": 0,
                  },
                ],
                "width": 100,
              },
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "cx": 80,
                "fill": "rgba(255,255,255, 0.3)",
                "r": "5",
              },
              "ref": [Function],
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <AnimatedComponent
                cx={70}
                fill="rgba(255,255,255, 0.3)"
                r="3"
              />,
              "style": Object {
                "bottom": 0,
                "height": 100,
                "left": 0,
                "position": "absolute",
                "transform": Array [
                  Object {
                    "translateY": 0,
                  },
                ],
                "width": 100,
              },
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "cx": 70,
                "fill": "rgba(255,255,255, 0.3)",
                "r": "3",
              },
              "ref": [Function],
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Circle
            cx="70"
            cy="70"
            fill="transparent"
            r="55"
            stroke="#efeff1"
            strokeWidth={4}
          />,
          "preserveAspectRatio": "xMidYMid meet",
          "style": Object {
            "bottom": -20,
            "height": 140,
            "left": -20,
            "position": "absolute",
            "width": 140,
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "cx": "70",
            "cy": "70",
            "fill": "transparent",
            "r": "55",
            "stroke": "#efeff1",
            "strokeWidth": 4,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Icon
            color="red"
            name="attention1"
            size={24}
            themeName="vibrantCyan"
          />,
          "style": Object {
            "alignItems": "center",
            "bottom": 36,
            "height": 24,
            "justifyContent": "center",
            "left": 38,
            "position": "absolute",
            "width": 24,
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "color": "red",
            "name": "attention1",
            "size": 24,
            "themeName": "vibrantCyan",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <View
            style={
              Object {
                "aspectRatio": 1,
                "overflow": "hidden",
                "width": 100,
              }
            }
          >
            <AnimatedComponent
              preserveAspectRatio="xMinYMin meet"
              style={
                Object {
                  "bottom": -23,
                  "height": 30,
                  "left": 0,
                  "position": "absolute",
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                  ],
                  "width": 420,
                }
              }
              viewBox="0 0 280 20"
            >
              <Path
                d="M 0 0
                   Q 35 -10 70 0
                   T 140 0
                   T 210 0
                   T 280 0
                   T 350 0
                   T 420 0
                   V 100
                   H 0
                   Z"
                transform="translate(0, 10)"
              />
            </AnimatedComponent>
            <AnimatedComponent
              preserveAspectRatio="xMinYMin meet"
              style={
                Object {
                  "bottom": -23,
                  "height": 30,
                  "left": 0,
                  "position": "absolute",
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                  ],
                  "width": 540,
                }
              }
              viewBox="0 0 360 25"
            >
              <Path
                d="M 0 0
                   Q 45 -15 90 0
                   T 180 0
                   T 270 0
                   T 360 0
                   T 450 0
                   T 540 0
                   V 100
                   H 0
                   Z"
                transform="translate(0, 15)"
              />
            </AnimatedComponent>
            <Svg
              preserveAspectRatio="xMidYMid meet"
              style={
                Object {
                  "bottom": -20,
                  "height": 140,
                  "left": -20,
                  "position": "absolute",
                  "width": 140,
                }
              }
            >
              <Circle
                cx="70"
                cy="70"
                fill="transparent"
                r="60"
                stroke="#ffffff"
                strokeWidth={24}
              />
            </Svg>
            <AnimatedComponent
              style={
                Object {
                  "bottom": 0,
                  "height": 100,
                  "left": 0,
                  "position": "absolute",
                  "transform": Array [
                    Object {
                      "translateY": 0,
                    },
                  ],
                  "width": 100,
                }
              }
            >
              <AnimatedComponent
                cx={80}
                fill="rgba(255,255,255, 0.3)"
                r="5"
              />
            </AnimatedComponent>
            <AnimatedComponent
              style={
                Object {
                  "bottom": 0,
                  "height": 100,
                  "left": 0,
                  "position": "absolute",
                  "transform": Array [
                    Object {
                      "translateY": 0,
                    },
                  ],
                  "width": 100,
                }
              }
            >
              <AnimatedComponent
                cx={70}
                fill="rgba(255,255,255, 0.3)"
                r="3"
              />
            </AnimatedComponent>
          </View>,
          <Svg
            preserveAspectRatio="xMidYMid meet"
            style={
              Object {
                "bottom": -20,
                "height": 140,
                "left": -20,
                "position": "absolute",
                "width": 140,
              }
            }
          >
            <Circle
              cx="70"
              cy="70"
              fill="transparent"
              r="55"
              stroke="#efeff1"
              strokeWidth={4}
            />
          </Svg>,
          <AnimatedComponent
            style={
              Object {
                "alignItems": "center",
                "bottom": 36,
                "height": 24,
                "justifyContent": "center",
                "left": 38,
                "position": "absolute",
                "width": 24,
              }
            }
          >
            <Icon
              color="red"
              name="attention1"
              size={24}
              themeName="vibrantCyan"
            />
          </AnimatedComponent>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              Array [
                <AnimatedComponent
                  preserveAspectRatio="xMinYMin meet"
                  style={
                    Object {
                      "bottom": -23,
                      "height": 30,
                      "left": 0,
                      "position": "absolute",
                      "transform": Array [
                        Object {
                          "translateX": 0,
                        },
                        Object {
                          "translateY": 0,
                        },
                      ],
                      "width": 420,
                    }
                  }
                  viewBox="0 0 280 20"
                >
                  <Path
                    d="M 0 0
                   Q 35 -10 70 0
                   T 140 0
                   T 210 0
                   T 280 0
                   T 350 0
                   T 420 0
                   V 100
                   H 0
                   Z"
                    transform="translate(0, 10)"
                  />
                </AnimatedComponent>,
                <AnimatedComponent
                  preserveAspectRatio="xMinYMin meet"
                  style={
                    Object {
                      "bottom": -23,
                      "height": 30,
                      "left": 0,
                      "position": "absolute",
                      "transform": Array [
                        Object {
                          "translateX": 0,
                        },
                        Object {
                          "translateY": 0,
                        },
                      ],
                      "width": 540,
                    }
                  }
                  viewBox="0 0 360 25"
                >
                  <Path
                    d="M 0 0
                   Q 45 -15 90 0
                   T 180 0
                   T 270 0
                   T 360 0
                   T 450 0
                   T 540 0
                   V 100
                   H 0
                   Z"
                    transform="translate(0, 15)"
                  />
                </AnimatedComponent>,
              ],
              <Svg
                preserveAspectRatio="xMidYMid meet"
                style={
                  Object {
                    "bottom": -20,
                    "height": 140,
                    "left": -20,
                    "position": "absolute",
                    "width": 140,
                  }
                }
              >
                <Circle
                  cx="70"
                  cy="70"
                  fill="transparent"
                  r="60"
                  stroke="#ffffff"
                  strokeWidth={24}
                />
              </Svg>,
              <AnimatedComponent
                style={
                  Object {
                    "bottom": 0,
                    "height": 100,
                    "left": 0,
                    "position": "absolute",
                    "transform": Array [
                      Object {
                        "translateY": 0,
                      },
                    ],
                    "width": 100,
                  }
                }
              >
                <AnimatedComponent
                  cx={80}
                  fill="rgba(255,255,255, 0.3)"
                  r="5"
                />
              </AnimatedComponent>,
              <AnimatedComponent
                style={
                  Object {
                    "bottom": 0,
                    "height": 100,
                    "left": 0,
                    "position": "absolute",
                    "transform": Array [
                      Object {
                        "translateY": 0,
                      },
                    ],
                    "width": 100,
                  }
                }
              >
                <AnimatedComponent
                  cx={70}
                  fill="rgba(255,255,255, 0.3)"
                  r="3"
                />
              </AnimatedComponent>,
            ],
            "style": Object {
              "aspectRatio": 1,
              "overflow": "hidden",
              "width": 100,
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "0",
              "nodeType": "class",
              "props": Object {
                "children": <Path
                  d="M 0 0
                   Q 35 -10 70 0
                   T 140 0
                   T 210 0
                   T 280 0
                   T 350 0
                   T 420 0
                   V 100
                   H 0
                   Z"
                  transform="translate(0, 10)"
                />,
                "preserveAspectRatio": "xMinYMin meet",
                "style": Object {
                  "bottom": -23,
                  "height": 30,
                  "left": 0,
                  "position": "absolute",
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                  ],
                  "width": 420,
                },
                "viewBox": "0 0 280 20",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "d": "M 0 0
                   Q 35 -10 70 0
                   T 140 0
                   T 210 0
                   T 280 0
                   T 350 0
                   T 420 0
                   V 100
                   H 0
                   Z",
                  "transform": "translate(0, 10)",
                },
                "ref": [Function],
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "1",
              "nodeType": "class",
              "props": Object {
                "children": <Path
                  d="M 0 0
                   Q 45 -15 90 0
                   T 180 0
                   T 270 0
                   T 360 0
                   T 450 0
                   T 540 0
                   V 100
                   H 0
                   Z"
                  transform="translate(0, 15)"
                />,
                "preserveAspectRatio": "xMinYMin meet",
                "style": Object {
                  "bottom": -23,
                  "height": 30,
                  "left": 0,
                  "position": "absolute",
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                  ],
                  "width": 540,
                },
                "viewBox": "0 0 360 25",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "d": "M 0 0
                   Q 45 -15 90 0
                   T 180 0
                   T 270 0
                   T 360 0
                   T 450 0
                   T 540 0
                   V 100
                   H 0
                   Z",
                  "transform": "translate(0, 15)",
                },
                "ref": [Function],
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <Circle
                  cx="70"
                  cy="70"
                  fill="transparent"
                  r="60"
                  stroke="#ffffff"
                  strokeWidth={24}
                />,
                "preserveAspectRatio": "xMidYMid meet",
                "style": Object {
                  "bottom": -20,
                  "height": 140,
                  "left": -20,
                  "position": "absolute",
                  "width": 140,
                },
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "cx": "70",
                  "cy": "70",
                  "fill": "transparent",
                  "r": "60",
                  "stroke": "#ffffff",
                  "strokeWidth": 24,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <AnimatedComponent
                  cx={80}
                  fill="rgba(255,255,255, 0.3)"
                  r="5"
                />,
                "style": Object {
                  "bottom": 0,
                  "height": 100,
                  "left": 0,
                  "position": "absolute",
                  "transform": Array [
                    Object {
                      "translateY": 0,
                    },
                  ],
                  "width": 100,
                },
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "cx": 80,
                  "fill": "rgba(255,255,255, 0.3)",
                  "r": "5",
                },
                "ref": [Function],
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <AnimatedComponent
                  cx={70}
                  fill="rgba(255,255,255, 0.3)"
                  r="3"
                />,
                "style": Object {
                  "bottom": 0,
                  "height": 100,
                  "left": 0,
                  "position": "absolute",
                  "transform": Array [
                    Object {
                      "translateY": 0,
                    },
                  ],
                  "width": 100,
                },
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "cx": 70,
                  "fill": "rgba(255,255,255, 0.3)",
                  "r": "3",
                },
                "ref": [Function],
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <Circle
              cx="70"
              cy="70"
              fill="transparent"
              r="55"
              stroke="#efeff1"
              strokeWidth={4}
            />,
            "preserveAspectRatio": "xMidYMid meet",
            "style": Object {
              "bottom": -20,
              "height": 140,
              "left": -20,
              "position": "absolute",
              "width": 140,
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "cx": "70",
              "cy": "70",
              "fill": "transparent",
              "r": "55",
              "stroke": "#efeff1",
              "strokeWidth": 4,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <Icon
              color="red"
              name="attention1"
              size={24}
              themeName="vibrantCyan"
            />,
            "style": Object {
              "alignItems": "center",
              "bottom": 36,
              "height": 24,
              "justifyContent": "center",
              "left": 38,
              "position": "absolute",
              "width": 24,
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "color": "red",
              "name": "attention1",
              "size": 24,
              "themeName": "vibrantCyan",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
